pages/nft-book-store/index.vue

```diff
@@ -31,6 +31,30 @@
         </tr>
       </table>
       <hr>
+      <template v-if="moderatedBookList.length">
+        <h2>Viewable listing</h2>
+        <table>
+          <tr>
+            <td>Class Id</td>
+            <td>Price in USD</td>
+            <td>Pending action</td>
+            <td>Sold</td>
+            <td>Remaining Stock</td>
+          </tr>
+          <tr v-for="b in moderatedBookList" :key="b.classId">
+            <td>
+              <NuxtLink :to="{ name: 'nft-book-store-status-classId', params: { classId: b.classId } }">
+                {{ b.classId }}
+              </NuxtLink>
+            </td>
+            <td>{{ b.prices?.map((p: any) => p.price).join(', ') }}</td>
+            <td>{{ b.pendingNFTCount }}</td>
+            <td>{{ b.sold }}</td>
+            <td>{{ b.stock }}</td>
+          </tr>
+        </table>
+      </template>
+      <hr>
```

Comment:
If `moderatedBookList` is empty, there will be 2 consecutive `<hr>`
```suggestion
        <hr>
      </template>
```